### ISSUE TITLE
export_to_phy bug with asymetrical window

### ISSUE THREAD

**Issue by Dradeliomecus** [SpikeInterface/spiketoolkit#439]
Hi,

I've used the `export_to_phy` method using as parameters `me_before=1.0, ms_after=3.0`, and when opening the output folder with phy, the result is a 4ms window, but the spike is perfectly centered at 2.0ms (whereas it should be at 1.0ms).

Here is the line used:
```
spiketoolkit.postprocessing.export_to_phy(data.recording, data.merged_sorting, "/some/folder", copy_binary=False, ms_before=1.0, ms_after=3.0, dtype=np.int16, recompute_info=True, save_property_or_features=False)
```

EDIT: While debugging, for faster export, I used the following line, and it now exports a 6ms window with the waveform centered at 3.0 ms:
```
spiketoolkit.postprocessing.export_to_phy(data.recording, data.merged_sorting, "/some/folder", compute_pc_features=False, compute_amplitudes=False, copy_binary=False, ms_before=1.0, ms_after=3.0, dtype=np.int16, recompute_info=False)
```

**Comment by alejoe91** [comment]
Not sure, but maybe phy assumes waveforms are centered? @rossant can you comment on that?

**Comment by Dradeliomecus** [comment]
Fixed with #457 (see issue #454)
